README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Join the chat at https://gitter.im/atnos-org/eff](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/atnos-org/eff?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Maven Central](https://img.shields.io/maven-central/v/org.atnos/eff_2.13.svg)](https://maven-badges.herokuapp.com/maven-central/org.atnos/eff_2.13)
+![CI](https://github.com/atnos-org/eff/workflows/CI/badge.svg)
```

Comment:
It would be better to use badge Markdown generated by GitHub Actions, isn't it? 🤔 

- <img width="255" alt="image" src="https://user-images.githubusercontent.com/612043/142050147-7ea50788-fd07-4339-bba1-2104fd8da0ae.png">


```suggestion
[![CI](https://github.com/atnos-org/eff/actions/workflows/ci.yml/badge.svg)](https://github.com/atnos-org/eff/actions/workflows/ci.yml)
```

Reply:
Thank you very much. you are right :pray: